client/src/main/js/service/AgendamentoService.js

```diff
@@ -39,43 +36,39 @@
             const inicio = reserva.inicio;
             const fim = reserva.fim;
 
+            console.log(inicio);
+            console.log(fim);
+            console.log('Reservas: ', reservasDia);
+
             for (let i = 0; i < reservasDia.length; i++) {
```

Comment:
`_.each(reservasDia, reservaDia => {})` ou `for (reservaDia in reservasDia) {}`. Dá uma olhada no [lodash](https://lodash.com/docs/4.17.4) pra saber de mais coisas legais q podem ser feitas :D